src/game.c

```diff
@@ -140,11 +140,11 @@ Gfx *gDisplayLists[2];
 Gfx *gCurrDisplayList;
 UNUSED s32 D_801211FC;
 Mtx *gHudMatrices[2];
-Mtx *gCurrHudMat;
+Mtx *gGameCurrMatrix;
 VertexList *gHudVertices[2];
-VertexList *gCurrHudVerts;
+VertexList *gGamecurrVertexList;
```

Comment:
```suggestion
VertexList *gGameCurrVertexList;
```
Will need to update all references as well.